Formula/rex.rb

```diff
@@ -12,6 +13,7 @@ class Rex < Formula
   end
 
   depends_on :perl => "5.16"
+  depends_on :macos => :mavericks
```

Comment:
I don't think we need this since users can install arbitrary versions of perl on lower macOS versions.